react-native-is-edge-to-edge/src/index.ts

```diff
@@ -0,0 +1,7 @@
+import { Platform, TurboModuleRegistry } from "react-native";
+
+// Edge-to-edge is only an Android problem
+const edgeToEdge =
```

Comment:
Having a bit of a rough time understanding what this package does. It gets the package and makes sure it is on Android? Why not have it on the main module? (I feel like there is a huge context thing that i am missing here)

Reply:
This package is for other libraries authors. By adding this very lightweight dependency (`react-native-is-edge-to-edge`, not `react-native-edge-to-edge`), they will be able to detect if edge-to-edge is enabled and adjust behaviors for their library to behave correctly.

For example, reanimated could adapt [`useAnimatedKeyboard` `isStatusBarTranslucentAndroid ` options](https://docs.swmansion.com/react-native-reanimated/docs/device/useAnimatedKeyboard#options-) automatically.

Reply:
Ah i see. Makes sense. Hadn't thought of other libraries.👍